README.md

```diff
@@ -27,3 +31,20 @@ We *do not* plan to keep permanent forks or to maintain components under this or
 ## what does this repository contain?
 
 This repository contain generic documentation which doesn't belong to specific repositories, maybe because, like this README, covers the organization, or because covers general topics.
+
+## Community, discussion, contribution, and support
+
+You can reach the maintainers of this project at:
```

Comment:
I'd add a note that this is a working group in the general, english sense of the word "A working group, or working party, is a group of experts working together to achieve specified goals. " (https://en.wikipedia.org/wiki/Working_group) and not a official k8s working group. While we want to cooperate with k8s, we don't have official endorsement, acknowledgement or anything else

Reply:
Done!